static/css/contact-us.less

```diff
@@ -3,10 +3,20 @@
    Contact us page.
    ========================================================================== */
 
-.helpful-terms_header {
-    .h5();
-    margin-right: unit(6px / @font-size, em);
-    vertical-align: middle;
+.helpful-terms {
+    &_header {
+        .h5();
+        margin-right: unit(6px / @font-size, em);
+        vertical-align: middle;
+    }
+
+    &_list {
```

Comment:
Fair point. I suppose if we didn't need the media query, the "standard" thing would've been to have both `list__unstyled` and `list__horizontal` in the class list. Fine to leave as-is.